GVFS/GVFS.UnitTests/Prefetch/DiffHelperTests.cs

```diff
@@ -66,11 +67,12 @@ public void CanParseDiffForwards()
         }
 
         // Parses Diff B => A
+        // Note that backward.txt does not contain a symlink entry so that we can get coverage on the includeSymLinks false case

```

Comment:
It might be worth testing `backward.txt` with `includeSymLinks` true and false.

Could you parameterize `DiffHelperTests` so and make `includeSymLinks` a member (that is set by the parameter)?

Reply:
(You can take a look at `ActiveEnumerationTests` for an example of how to do this)

Reply:
That's a nice pattern, thanks for the pointer.